integration

```diff
@@ -443,6 +443,7 @@
   "jcgoette/weight_gurus_homeassistant",
   "jcwillox/hass-auto-backup",
   "JeffSteinbok/hass-dreo",
+  "jekalmin/extended_openai_conversation",
```

Comment:
Are you still woking on this?
It does not look like there has been any activity on your github account the past half year.

Reply:
Sorry for late response.
I just got back and started working on it.